src/platform-includes/profiling/automatic-instrumentation-intro/_default.mdx

```diff
@@ -0,0 +1,7 @@
+```bash
+# Using yarn
+yarn add @sentry/browser
+
+# Using npm
+npm install --save @sentry/browser
+```
```

Comment:
Can we use code tabs for these instead? Also, I think it's preferred to put npm first: https://github.com/getsentry/sentry-docs/pull/8362

```suggestion
```bash {tabTitle:npm}
npm install --save @sentry/browser
```

```bash {tabTitle:yarn}
yarn add @sentry/browser
```
```

Reply:
ahh whoa sorry github really doesn't like fenced code blocks in suggestions.

If you edit this comment you can grab what it should be


```bash {tabTitle:npm}
npm install --save @sentry/browser
```

```bash {tabTitle:yarn}
yarn add @sentry/browser
```

Reply:
Oh yes, updated!